prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -36,6 +36,9 @@ presubmits: # runs on PRs
         preset-sa-gke-kyma-integration: "true"
         preset-gc-compute-envs: "true"
         preset-gc-project-env: "true"
+        preset-build-pr: "true"
+        preset-docker-push-repository: "true"
+        preset-dind-enabled: "true"
       decorate: true
```

Comment:
Please update the `bootstrap` image tag to make sure `dig` is included. Earlier versions of the current tag `0.0.1` did not have it and if it is cached in the cluster, this leads to problems. You can use this tag instead: `v20181119-afd3fbd`.

Reply:
I've updated to eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181121-f3ea5ce